src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,16 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
         //TODO: add corresponding log message about method login was called
```

Comment:
Please do not send a task for checking without looking at the checklist. Next time I will reject it without checking 
<img width="461" alt="Screen Shot 2022-10-13 at 8 52 05 PM" src="https://user-images.githubusercontent.com/90692308/195670120-1c2307ac-4f0b-4733-bae9-707502234cc6.png">